datastore/postgres/indexer_store.go

```diff
@@ -39,12 +38,14 @@ var _ indexer.Store = (*IndexerStore)(nil)
 //
 // All the other exported methods live in their own files.
 type IndexerStore struct {
-	pool *pgxpool.Pool
+	pool     *pgxpool.Pool
+	scanners map[string]int64
```

Comment:
this may be a decent candidate for `map[unique.Handle[string]]int64` since I think this map will be rather static once it's initially populated